.changeset/wise-houses-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/blocks-admin": minor
+---
+
+Add `visibleOrderedBlocksForState` to `createCompositeBlocks` factory options base
```

Comment:
This changeset should explain what this option is used for.

Reply:
improved changeset